conf/alluxio-site.properties.template

```diff
@@ -13,8 +13,8 @@
 # Details about all configuration properties https://docs.alluxio.io/os/user/stable/en/reference/Properties-List.html
 
 # Common properties
-# alluxio.master.hostname=localhost
-# alluxio.master.mount.table.root.ufs=${alluxio.work.dir}/underFSStorage
+# alluxio.coordinator.hostname=localhost
```

Comment:
the property key string did not get renamed. should it be left alone for now and only focus on the java package rename?

Reply:
good call, make a new PR focused on package name: https://github.com/Alluxio/alluxio/pull/17429
I will close this one